ipfix-flow-exporter/bpf_ipfix_egress_user.c

```diff
@@ -132,37 +108,14 @@ int main(int argc, char **argv)
         return EXIT_FAILURE;
     }
 
-    if(attach_tc_egress_filter) {
-	if(chain == true){
-            if (tc_chain_bpf(bpf_map_file_path, filename, egress) != 1) {
-                fprintf(stderr, "ERR: tc chain egress failed \n");
-                close_logfile();
-                 return EXIT_FAILURE;
-            }
-	}
-        else {
-            if (tc_attach_filter(if_name, filename, EGRESS, egress) != 1) {
-                fprintf(stderr, "ERR: TC EGRESS filter attach failed\n");
-                close_logfile();
-                return EXIT_FAILURE;
-            }
-       }
-       if (populate_egress_fds() == EXIT_FAILURE) {
-            fprintf(stderr, "ERR: Fetching TC EGRESS maps failed\n");
-            close_logfile();
-            return EXIT_FAILURE;
-       }
-       if (remote_ip == NULL) {
-                log_info("Remote IP is not configured by user, so configuring localhost as remote_ip");
-                remote_ip = "127.0.0.1" ;
-       }
+    if (populate_egress_fds() == EXIT_FAILURE) {
+        fprintf(stderr, "ERR: Fetching TC EGRESS maps failed\n");
+        close_logfile();
+        return EXIT_FAILURE;
     }
-
-    if (remove_egress_tc_filter) {
-        log_debug("TC remove egress filters on device %s",
-                   if_name);
-        tc_remove_filter(if_name, EGRESS);
-        return EXIT_SUCCESS;
+    if (remote_ip == NULL) {
+        log_info("Remote IP is not configured by user, so configuring localhost as remote_ip");
+        remote_ip = "127.0.0.1" ;
```

Comment:
"120.0.0.1" is only one of the loopback addresses that "localhost" resolves to
("::" being the IPv6 loopback address).  If you mean "IPv4 loopback address" then say that, but this is different from "localhost" which has *2* addresses. 

Separately, I don't like defaulting to IPv4 only.  I'd rather have no default (force it to be configured) than an IPv4-only default.  If it defaulted to both IPv4 and IPv6 that would be even better.